### DISCUSSION TITLE
Orca causing scraping issue due to somne initial missing commands

### DISCUSSION THREAD

**Discussion by enewbold** [OrcaSlicer/OrcaSlicer#1486] [General]
Every time I print an Orca-sliced model on the Flsun V400, after laying the initial perimeter bead of filament, there is a loud **scraping** sound on the build plate as the nozzle moves into the starting print position.  This only happens with Orca.  I sliced a test cube in both Prusa 2.6 and in Orca 1.6.3 for comparison.  The code Prusa produces does not scratch the build plate; the one from Orca does scratch the build plate.

I then examined the gcode from both slicers for this test cube (and some other models), and I noticed one thing right off the bat:  After laying the initial perimeter bead of filament, but before moving to the starting print position, Prusa issues a retract command and Orca does not.  To my simple mind, this seems to be the issue.  

Not being a gcode expert, however, I don't know how to fix this, and in the Orca interface I don't see any way to address the issue.  A quick fix to this in the next build would be greatly appreciated.  Thank you.

**Comment by Glenn1963** [comment]
I don't think that should be happening, and isn't on my V400. My initial perimeter prints at a Z height of 0.3 which should be plenty to keep the nozzle off the bed. 

Is your bed levelled and does your machine start code or start macro include the line
`BED_MESH_PROFILE LOAD=default`

Having said that, to more directly address your request...

I assume your start machine code includes the purge line (or calls a macro that includes the line); that code should look something like this:
```
G1 F3000 Z1 ; move to 1mm above bed
G1 X-150 Y0 Z0.4 ;move in to position to start your purge line
G92 E0 ;zero your extruder
G3 X0 Y-130 I150 Z0.3 E30 F2000 ;purge line
G92 E0 ;reset and zero extruder after purge
```
So after the G3 purge line you could add a Z lift line, like
`G1 Z10 ; Lift head `
would lift the head up to 10mm before the actual print started.

**Reply by enewbold** [reply]
Yes, my bed is level and zeroed and my start machine code looks like just that.  But, I will add the suggested G1 Z10 line to it to see what happerns.  Thank you.

**Comment by Charlotte-br560** [comment]
It sounds like there might be some initial commands missing in Orca causing scraping issues.
